### PR TITLE
Basic readme setup for localization plugin

### PR DESCRIPTION
**Pull request by langsmith**
Adds basic docs/links for localization release https://github.com/mapbox/mapbox-plugins-android/issues/323

Related:

https://github.com/mapbox/android-docs/issues/321

https://github.com/mapbox/mapbox-android-demo/issues/629

### CODE REVIEW

File: plugin-localization/README.md [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+repositories {
+    mavenCentral()
+    maven { url 'https://maven.google.com' }
+}
+
+...
+
+// In the app build.gradle file
+dependencies {
+    compile 'com.mapbox.mapboxsdk:mapbox-android-plugin-localization:0.1.0'
```
Comment (cammace) [inline plugin-localization/README.md]: implementation

File: plugin-localization/README.md [hunk 2]
```diff
@@ -1,3 +1,65 @@
...
+}
+```
+
+The location layer plugin is published to Maven Central and nightly SNAPSHOTs are available on Sonatype:
+
+```gradle
+// In the root build.gradle file
+repositories {
+    mavenCentral()
+    maven { url 'https://maven.google.com' }
```
Comment (cammace) [inline plugin-localization/README.md]: simplify to `google()`

Comment (langsmith) [reply plugin-localization/README.md]: Fixed

File: plugin-localization/README.md (hunk 1 above)
Comment (langsmith) [reply plugin-localization/README.md]: Fixed

**Comment by langsmith** [comment]
Should be good to go here @cammace

File: plugin-localization/README.md (hunk 2 above)
Comment (cammace) [reply plugin-localization/README.md]: This is still not fixed?

**Pull request merged**